locales/nl-NL/src/structure.ts

```diff
@@ -137,13 +137,13 @@ export default removeUndefinedLocaleResources({
   'buttons.split-pane-close-group-button.title': 'Sluit paneelgroep',
 
   /** The label used in the changes inspector for the from selector */
-  'changes.from.label': undefined, // 'From'
+  'changes.from.label': 'Van',
   /* The label for the history tab in the changes inspector*/
-  'changes.tab.history': undefined, // 'History'
+  'changes.tab.history': 'Geschiedenis',
   /* The label for the review tab in the changes inspector*/
-  'changes.tab.review-changes': undefined, // 'Review changes'
+  'changes.tab.review-changes': 'Wijzigingen beoordelen',
   /** The label used in the changes inspector for the to selector */
-  'changes.to.label': undefined, // 'To'
+  'changes.to.label': 'Naar',
```

Comment:
TODO: I want to double check other languages if this should be "Aan".

Reply:
Did you figure this one out, or should I release as-is? You can always open a PR to change it in the future, for what it’s worth

Reply:
I'm missing some context. Is it 'to' a person (send) or 'to' like move an element in the UI?

https://github.com/sanity-io/sanity/blob/d6dde60f6e1ad116d94e84643d7661f778d59c4b/packages/sanity/src/structure/i18n/resources.ts#L157